src/styles/style.css

```diff
@@ -0,0 +1,3 @@
+@import "./normalize.css";
+@import "./blocks/stars-raiting.css";
```

Comment:
Css file name should be equal to BEM block name: stars block = stars.css file name